Solutions/PaloAltoCDL/Hunting Queries/PaloAltoCDLRareApplicationLayerProtocol.yaml

```diff
@@ -0,0 +1,26 @@
+id: 0a18756a-5123-11ec-bf63-0242ac130002
+name: PaloAlto - Rare application layer protocols
+description: |
+  'Query shows Rare application layer protocols'
+severity: Low
+requiredDataConnectors:
+  - connectorId: PaloAltoCDL
+    dataTypes:
+      - PaloAltoCDLEvent
+tactics:
+  - InitialAccess
+relevantTechniques:
+  - T1190
+  - T1133
+query: |
+  PaloAltoCDLEvent
+  | where TimeGenerated > ago(24h)
+  | where isnotempty(NetworkApplicationProtocol) 
+  | summarize ApplicationLayerProtocol = count() by NetworkApplicationProtocol
+  | sort by ApplicationLayerProtocol asc

```

Comment:
How do we determine what is rare. Did we also wanted to do a top 5 or top 10 here.